modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/notifications/model/NotificationPreference.kt

```diff
@@ -0,0 +1,75 @@
+package au.com.shiftyjelly.pocketcasts.settings.notifications.model
+
+/**
+ * This enum defines the preferences we keep record on
+ */
+internal enum class NotificationPreferences {
```

Comment:
The naming feels a bit off here. Something like `NotificationPreferenceType` or `NotificationTypePreferenceId` might be a bit clearer and more descriptive.